Lookup CSV output fix

Shellyber opened the pull request:
## Status
- [x] Ready

## Related Issues
fixes: [CIAC-11129](https://jira-dc.paloaltonetworks.com/browse/CIAC-11129)

## Description
Added the output: LookupCSV.Result to the yml

ShahafBenYakir reviewed (commented):
Why have we ADDED the new output to the yml instead of fixing the wrong one? it does not work anyway so no real BC break here...

Shellyber commented:
> Why have we ADDED the new output to the yml instead of fixing the wrong one? it does not work anyway so no real BC break here...

Didn't want to break BC in case someone did use it.
Updated the yml

Shellyber commented:
@ShahafBenYakir Pre-commit fails on `[ERROR] Packs/CommonScripts/Scripts/LookupCSV/LookupCSV.yml: [BC101] - The following output keys: LookupCSV.result. Has been removed, please undo`
Will need to force merge this.

ShahafBenYakir commented on Packs/CommonScripts/ReleaseNotes/1_15_31.md:
```diff
@@ -0,0 +1,7 @@
+##### LookupCSV
+- Updated the Docker image to: *demisto/python3:3.11.9.103066*.
+
+- Added *LookupCSV.Result* to the context output.
```
```suggestion
- Fixed an issue where the script results were added to *LookupCSV.Result* instead of *LookupCSV.Result* in the context.
```

Shellyber commented:
@ShahafBenYakir I ran nightly as well - the validation passed - https://gitlab.xdr.pan.local/xdr/cortex-content/content/-/jobs/7066383
Please force when you have a chance
